kratos/mpi/includes/mpi_communicator.h

```diff
@@ -815,6 +815,12 @@ template<OperationType TOperation> struct Operation
         return true;
     }
 
+    bool SynchronizeNonHistoricalVariable(Variable<bool> const& rThisVariable) override
+    {
+        SynchronizeNonHistoricalVariable<bool,bool>(rThisVariable);
```

Comment:
this should also be removed, I am quite sure it breaks the MPI-compilation

Reply:
Yes this is the old version... Probably the merge went wrong, an updated version of this function already exists in master.

Reply:
This is outdated, I removed it from the files (I restored master versions)